### PR TITLE
Improve traceability of `ITimerMsg`

### PR DESCRIPTION
**Pull request by Aaronontheweb**
## Changes

This is mostly a [Phobos](https://phobos.petabridge.com/)-oriented improvement, aimed at making it easier to keep track of scheduled metrics in systems that use `IWithTimer` heavily. Specifically, we now include the content of the message being scheduled inside the payload so we can answer the question "what was scheduled?" without a lot of overhead.

This doesn't affect the public API of Akka.NET at all.

## Checklist

For significant changes, please ensure that the following have been completed (delete if not relevant):

* [x] This change follows the [Akka.NET API Compatibility Guidelines](https://getakka.net/community/contributing/api-changes-compatibility.html).
* [x] This change follows the [Akka.NET Wire Compatibility Guidelines](https://getakka.net/community/contributing/wire-compatibility.html).
* [x] I have [reviewed my own pull request](https://getakka.net/community/contributing/index.html#review-your-own-pull-requests).

### CODE REVIEW

File: src/core/Akka/Actor/Scheduler/TimerScheduler.cs [hunk 1]
```diff
@@ -41,36 +41,39 @@ public Timer(object key, object msg, bool repeat, int generation, ICancelable ta
             }
         }
 
-        public interface ITimerMsg
+        public interface ITimerMsg : IWrappedMessage, INoSerializationVerificationNeeded
```
Comment (Aaronontheweb) [inline src/core/Akka/Actor/Scheduler/TimerScheduler.cs:44]: `IWrappedMessage` is how we can get access to the contents of the timer and record it via Phobos metrics.

File: src/core/Akka/Actor/Scheduler/TimerScheduler.cs [hunk 2]
```diff
@@ -41,36 +41,39 @@ public Timer(object key, object msg, bool repeat, int generation, ICancelable ta
...
-            public TimerMsg(object key, int generation, TimerScheduler owner)
+            public TimerMsg(object key, int generation, TimerScheduler owner, object message)
             {
-                this.Key = key;
-                this.Generation = generation;
-                this.Owner = owner;
+                Key = key;
+                Generation = generation;
+                Owner = owner;
+                Message = message;
```
Comment (Aaronontheweb) [inline src/core/Akka/Actor/Scheduler/TimerScheduler.cs:62]: We're not even going to access this value directly in the main timer use-case, so none of that code path is affected.

All of these messages types are `internal` so there's no public API impact.

**Review by Aaronontheweb — COMMENTED** [review]
Detailed my changes - going to do a local POC with Phobos and see if this resolves my issue there.

**Review by Arkatufus — APPROVED** [review]
LGTM

**Pull request merged**